docs/introduction.md

```diff
@@ -59,3 +59,9 @@ After the promise is settled, a second action will be dispatched. If the the pro
   }
 }
 ```
+
+The default promise status suffix can be imported from this module.
+
+```js
+import { PENDING, FULFILLED, REJECTED } from 'redux-promise-middleware'
```

Comment:
This is incorrect as you've named the constants with a prefix `DEFAULT_`

I actually think the prefix is a little unnecessarily verbose and it might be better to just call them PENDING, FULFILLED and REJECTED like this documentation comment suggests.

E.g. The implementation you've added would be:
```js
import { DEFAULT_PENDING, DEFAULT_FULFILLED, DEFAULT_REJECTED } from 'redux-promise-middleware'
```